apiserver/facades/agent/secretsmanager/secrets.go

```diff
@@ -26,10 +27,7 @@ import (
 	"github.com/juju/juju/state"
 )
 
-// For testing.
-var (
-	GetProvider = secretsprovider.Provider
-)
+var logger = loggo.GetLoggerWithLabels("juju.apiserver.secretsmanager", corelogger.SECRETS)
```

Comment:
I don't think we need this. Coz we have SecretsManagerAPI.logger.

Reply:
Yep you're right